README.md

```diff
@@ -71,7 +71,10 @@ readme = remark().use(plugin, {
 
 ### Output
 
-This will add a "Contributors" section to your document if one is not already present. If one is present, a list of contributors will be added to that section.
+This will add a `Contributors` section to your document if one is not already
+present and you set [`appendIfMissing`](#options.appendIfMissing) to `true` in
+the options. If such section already is present, a list of contributors will be
+added to that section.
```

Comment:
I think this line still needs to be changed to note the breaking change?

Reply:
how so?

Reply:
I believe you first proposed false as a default (back compat), but it later got changed to a breaking change?
The above section seems to indicate.

Or, maybe this section is just hard to read, I’m not sure what the default is anymore from reading this? 🤔

Reply:
Agree that it's hard to read. Suggestion:

This plugin will add or update a list of contributors in a `Contributors` section. If such a section is not present, nothing happens by default. To have the plugin add a new section to the end of the document, set [`appendIfMissing`](#options.appendIfMissing) to `true`.

Reply:
Also, the markdown example below that paragraph is incorrect.